README.md

```diff
@@ -22,3 +22,6 @@ In order to start using `gostats`, import it into your project with:
 ```go
 import "github.com/lyft/gostats"
 ```
+
+## Timers
+It is not obvious but due to a but all gostats timers output timespans in *microseconds*, not milliseconds as the stat string itself suggests.
```

Comment:
> It is not obvious but due to a but all gostats timers output timespans in *microseconds*, not milliseconds as the stat string itself suggests.

Maybe something like:

The `gostats` timers output timespans in *microseconds*, not milliseconds as the stat string itself suggests.  This is not obvious and clearly incorrect, but changing it would break backwards compatibility.